### DISCUSSION TITLE
Ideas and suggestions

### DISCUSSION THREAD

**Discussion by Pradumnasaraf** [Pradumnasaraf/open-source-with-pradumna#26] [Ideas]
#### Thank you for your support. Feel free to drop your ideas and suggestions with supporting docs and images below

**Comment by vikasganiga05** [comment]
Hi Pradumnasaraf, The [How-to guides](https://github.com/Pradumnasaraf/open-source-with-pradumna/tree/main/pages/How-to) are not completed right now. So, my suggestion is to add a badge like the below one as a placeholder instead of a link to the empty markdown file :point_down: 

![Work In Progress](https://img.shields.io/badge/Markdown%20Guide-WIP-yellow)
![Work In Progress](https://img.shields.io/badge/Markdown%20Guide-WIP-yellow?style=flat-square)
![Work In Progress](https://img.shields.io/badge/Markdown%20Guide-WIP-yellow?style=for-the-badge)

#### Screenshot

![image](https://user-images.githubusercontent.com/74750414/169107986-61b97c83-fa3a-4418-a91d-1c542a961b3d.png)

**Reply by Pradumnasaraf** [reply]
Great point, thank you for the suggestion.

What's your thought on adding them inside the markdown, so that we don't have to unlink them?

**Reply by vikasganiga05** [reply]
Yeah! that sounds great :+1:

**Reply by Pradumnasaraf** [reply]
Thank you!.
Would you like to raise an issue for that?, also can open a PR.

**Reply by vikasganiga05** [reply]
Yes, I will work on that :slightly_smiling_face: Thanks!

**Reply by Pradumnasaraf** [reply]
Thank you!, Vikash. Do let me know if you have any questions.

**Reply by vikasganiga05** [reply]
Which badge should I add 1, 2, or 3?

**Reply by Pradumnasaraf** [reply]
I find 3rd one is a little bigger and has much clearer text, what are your thoughts on this?

**Reply by vikasganiga05** [reply]
Yes, I taught of adding the 3rd one. But we can add a banner like :point_down: 

![image](https://user-images.githubusercontent.com/74750414/169114265-1348fa40-d66c-401d-b65f-72e495740b6c.png)

With the text `Work In Progress` What do you think of this idea?

**Reply by vikasganiga05** [reply]
Please have a look at added examples banners on the issue &rarr; https://github.com/Pradumnasaraf/open-source-with-pradumna/issues/28

**Reply by Pradumnasaraf** [reply]
Indeed, banner looks more robust, thank you for making it

**Reply by vikasganiga05** [reply]
Thank You :pray:
